cmd/backup-manager/app/export/manager.go

```diff
@@ -262,6 +263,8 @@ func (bm *BackupManager) performBackup(backup *v1alpha1.Backup, db *sql.DB) erro
 			Message: backupErr.Error(),
 		})
 		errs = append(errs, uerr)
+		// just delete backupFullPath since it will never be used except for debug
```

Comment:
If it's useful for debug, can we keep it just in case?

Reply:
addressed in https://github.com/pingcap/tidb-operator/pull/2897/commits/df73f02c027c04d97c78084e29bb5bb59b1bb7f3